mobile/src/home-view.js

```diff
@@ -315,7 +317,7 @@ class HomeView extends PureComponent {
       color: '#364247',
       textAlignVertical: 'top',
       maxHeight: 100,
-      height: Math.max(35, inputHeight),
+      height: 35,
```

Comment:
This looks like we can now move this style to StyleSheet.create, since it is only constants, now.